linera-base/src/identifiers.rs

```diff
@@ -921,6 +942,7 @@ impl<A> ApplicationId<A> {
 enum SerializableAccountOwner {
     Reserved(u8),
     Address32(CryptoHash),
+    Address20([u8; 20]),
```

Comment:
Tried using `alloy_primitives::Address` here but the RPC format generation would panic when it tried to deserialize the generated value into `alloy_primitivies::Address` .